### PR TITLE
feat: Migrated to npm release of riot-tmpl fork.

### PR DESCRIPTION
**Pull request by alexgrozav**

### CODE REVIEW

File: packages/workflow/package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "lodash.merge": "^4.6.2",
     "lodash.set": "^4.3.2",
     "luxon": "^2.3.0",
-    "riot-tmpl": "github:n8n-io/tmpl",
+    "riot-tmpl": "npm:@n8n_io/riot-tmpl@^1.0.1",
```
Comment (ivov) [inline packages/workflow/package.json:61]: Why not as usual?
```suggestion
    "@n8n_io/riot-tmpl": "^1.0.1",
```

**Review by ivov — COMMENTED** [review]
This works in that...
- It installs our fork v1.0.1, but I do not know how to properly test myself that it works with the nightly build. Assuming that it works because of [this](https://github.com/n8n-io/n8n/actions/runs/2547554154).
- Tests in `/workflow` package pass.
- `window` becomes `{}` in editor when manually checking.

Can you please add `/* eslint-disable @typescript-eslint/no-unsafe-return */ ` to the top of `Logger.ts` so that the build passes?

File: packages/workflow/package.json (hunk 1 above)
Comment (alexgrozav) [reply packages/workflow/package.json]: Yeah, good point. We won't revert back to riot-tmpl. Updated imports.

**Comment by alexgrozav** [comment]
@ivov Looks like @MiloradFilipovic tried it out and the nightly build works with the changes.

**Comment by janober** [comment]
Great, thanks a lot for fixing it. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.184.0`

**Pull request merged**